utils/litemap/Cargo.toml

```diff
@@ -4,7 +4,7 @@
 
 [package]
 name = "litemap"
-version = "0.2.2"
+version = "0.3.0"
```

Comment:
because litemap had a lot of changes

From https://github.com/unicode-org/icu4x/issues/1546#issuecomment-1022718266

> * litemap: 0.2.2, [3c019c9](https://github.com/unicode-org/icu4x/commit/3c019c9c1ce1476a6e41325e22fbce1447dab127). New APIs only, might be worth larger bump anyway

Reply:
@Manishearth clarified that the change in serialization behavior pushes this over the edge to be considered for a major bump.